setup.py

```diff
@@ -52,37 +49,7 @@ def run(self):
 
 
 setup(name = "plumbum",
-    version = version_string,  # @UndefinedVariable
-    description = "Plumbum: shell combinators library",
-    author = "Tomer Filiba",
-    author_email = "tomerfiliba@gmail.com",
-    license = "MIT",
-    url = "https://plumbum.readthedocs.io",
-    packages = ["plumbum", "plumbum.cli", "plumbum.commands", "plumbum.machines", "plumbum.path", "plumbum.fs", "plumbum.colorlib"],
-    package_data={
-        "plumbum.cli":["i18n/*/LC_MESSAGES/*.mo"]
-    },
-    platforms = ["POSIX", "Windows"],
-    provides = ["plumbum"],
-    keywords = "path, local, remote, ssh, shell, pipe, popen, process, execution, color, cli",
     cmdclass = {'test':PyTest,
```

Comment:
I'm okay with killing off 'test' and `test_requires` too, by the way; Python is deprecating `setup.py test`. The only correct way to run the tests is via `pytest`.